code/game/objects/items/license.txt

```diff
@@ -0,0 +1,3 @@
+icons/obj/plushies.dmi's icon state of secdrake and meddrake by Mizartz. It has been licensed under the CC BY-NC-SA 3.0 license.
```

Comment:
place this into the icons/ instead of the object

Reply:
Moved to icons/